nttt/tidyup.py

```diff
@@ -43,33 +43,33 @@ def increase_indent(self, flow=False, indentless=False):
                                width=1000000)
 
 
-def fix_step(src, lang, dst, disable, logging):
-    content, suggested_eol = get_file(src)
-    content = content.replace("\---", "---")
-    content = content.replace("## ---", "---")
-    content = content.replace("--- hints ---", "--- hints ---\n")
-    content = content.replace(" --- hint --- ", "--- hint ---\n")
-    content = content.replace(" --- /hint ---", "\n--- /hint ---\n")
-    content = content.replace(" --- /hints ---", "--- /hints ---")
-    content = content.replace('{: target = " blank"}', '{:target="blank"}')
-    content = content.replace("\n` ", "\n`")
+def fix_md_step(src, lang, dst, disable, logging):
+    md_content, suggested_eol = get_file(src)
```

Comment:
Added `_md_` to variables, function names and arguments to make it clear that the text the file content is treated as Markdown code